app/testonly/TestUsersController.scala

```diff
@@ -59,7 +59,7 @@ final case class TestUserForm(
     isOnIA                       = isOnIA,
     authorityId                  = authorityId.map(AuthorityId.apply).getOrElse(AuthorityId.random),
     affinityGroup                = AffinityGroup(affinityGroup),
-    confidenceLevel              = if (hasOverTwoHundred) L200.level else L100.level,
+    confidenceLevel              = if (hasOverTwoHundred) L200.level else L50.level,
```

Comment:
Why was this changed to L50?

Reply:
L100 doesn't exist in the latest release of hmrc's auth-client (I think confidence levels are generally on the way out?)

this controller sets up test users for manual testing so won't affect business logic

Reply:
ok